build/.goreleaser.yml

```diff
@@ -0,0 +1,106 @@
+builds:
+  - id: nri-nix
+    main: ./src
+    binary: nri-kafka
+    gobinary: "go1.9.7"
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+  - id: nri-win
+    main: ./src
+    binary: nri-kafka
+    gobinary: "go1.9.7"
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - windows
+    goarch:
+      - 386
+      - amd64
+    hooks:
+      pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "kafka"
+
+nfpms:
+  - id: nri-deb
+    builds:
+      - nri-nix
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure kafka Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from kafka components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - deb
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      "kafka-config.yml.sample": "/etc/newrelic-infra/integrations.d/kafka-config.yml.sample"
+      "CHANGELOG.md": "/usr/share/doc/nri-kafka/CHANGELOG.md"
+      "README.md":  "/usr/share/doc/nri-kafka/README.md"
+      "LICENSE": "/usr/share/doc/nri-kafka/LICENSE"
+    config_files:
+      "kafka-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/kafka-definition.yml"
+    dependencies:
+      - "newrelic-infra"
+
```

Comment:
the current package has a dependency with jmx , i just see that for windows in the new pipeline.
I think it should be listed here